src/test/java/uk/nhs/ctp/auditFinder/AWSAuditFinderTest.java

```diff
@@ -162,6 +141,68 @@ public void findAll_withFailedParsing_shouldFail() throws IOException {
     auditFinder.findAll(76L);
   }
 
+  @Test
+  public void findEncounters_nullClient_shouldFail() {
+    var auditFinder = new AWSAuditFinder(null, authService, objectMapper);
+
+    expectedException.expect(IllegalArgumentException.class);
```

Comment:
Same as the other null client test, this seems odd.
Also, the IllegalArgumentException thrown is wrong, as it's an argument of the constructor which was invalid, not this call.